network/p2p/gossip/gossip.go

```diff
@@ -34,6 +34,16 @@ const (
 	unsentType = "unsent"
 	sentType   = "sent"
 
+	// omitted indicate that the gossipable element was not added to the set due to some reason.
+	// for sent message, we'll use notReceive below.
+	omittedLabel = "omitted"
```

Comment:
nit: In other parts of the mempool code we use the term "dropped" for this

Reply:
I'll rename to dropped.